metron-platform/metron-elasticsearch/src/test/java/org/apache/metron/elasticsearch/integration/ElasticsearchIndexingIntegrationTest.java

```diff
@@ -102,11 +102,16 @@ public void setAdditionalProperties(Properties topologyProperties) {
     topologyProperties.setProperty("es.clustername", "metron");
     topologyProperties.setProperty("es.port", "9300");
     topologyProperties.setProperty("es.ip", "localhost");
-    topologyProperties.setProperty("indexing.writer.class.name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
+    topologyProperties.setProperty("indexing_writer_class_name", "org.apache.metron.elasticsearch.writer.ElasticsearchWriter");
   }
 
   @Override
   public String cleanField(String field) {
     return field;
   }
```

Comment:
The latest commits move the templates to their respective modules (metron-enrichment and metron-elasticsearch).  They are copied to metron-deployment at build time with copy-resources.  Let me know if this satisfies the separation of concerns concern.